src/simulation/m_time_steppers.fpp

```diff
@@ -723,6 +841,26 @@ contains
             call s_pressure_relaxation_procedure(q_cons_ts(1)%vf)
         end if
 
+        if (adv_n .and. alter_alpha) then
+            !$acc parallel loop collapse(3) gang vector default(present)
```

Comment:
This (and the other copies of it) should probably be abstracted into a subroutine to keep the time-stepping subroutine cleaner

Reply:
agreed